monai/networks/blocks/convolutions.py

```diff
@@ -60,10 +61,11 @@ def __init__(
         out_channels: int,
         strides: int = 1,
         kernel_size: Union[Sequence[int], int] = 3,
-        act=Act.PRELU,
-        norm=Norm.INSTANCE,
-        dropout=None,
+        act: Optional[Union[tuple, str]] = Act.PRELU,
+        norm: Optional[Union[tuple, str]] = Norm.INSTANCE,
```

Comment:
I think it's not `Optional`, please double confirm.
Thanks.

Reply:
Yes, Norm layer should not be optional, I've changed. Whereas, activation layer should be optional according to 1) the original comments in the code, 2) sometimes we do not need this layer, or we may do some manipulations for features after the norm layer first, and then do the activations, just as what resnet's resudial block did  in the downsample part ([https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py](url))